ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -19321,6 +19324,9 @@ void HqlCppTranslator::pickBestEngine(WorkflowArray & workflow)
         }
         setTargetClusterType(HThorCluster);
         DBGLOG("Thor query redirected to hthor instead");
+        Owned<IWUException> we = wu()->createException();
+        we->setSeverity(SeverityInformation);
+        we->setExceptionMessage("Thor query redirected to hthor.  Set #option ('pickBestEngine', 0) to override.");
```

Comment:
trivial: Is double space between sentences in this and prev. setExceptionMessage statement intentional ?

Reply:
It was, but I can remove if not desired.  I'll defer to Richard's judgement.